sqlmesh/dbt/dbtcontext.py

```diff
@@ -0,0 +1,151 @@
+from __future__ import annotations
```

Comment:
Not sure I like the name of the module. Why not just `context.py`?

Reply:
I thought about naming it context, but went with dbtcontext since that's the name of the class. I don't have a strong opinion either way. Let me know what you prefer.

Reply:
`context.py` is fine